phoenix-core/src/it/java/org/apache/phoenix/end2end/ExtendedQueryExecIT.java

```diff
@@ -51,7 +51,7 @@ public void testToDateFunctionBind() throws Exception {
         Date date = new Date(1);
         String tenantId = getOrganizationId();
 
-        initATableValues(tenantId, getDefaultSplits(tenantId),date, ts);
+        initATableValues("aTable", tenantId, getDefaultSplits(tenantId),date, ts, getUrl());
```

Comment:
If you need to pass the atable as name then use the static constant defined in BaseTest.